master/internal/provisioner/aws.go

```diff
@@ -212,11 +207,6 @@ func (c *awsCluster) terminate(ctx *actor.Context, instanceIDs []string) {
 		return
 	}
 
-	ctx.Log().Infof(
```

Comment:
Same question here. It seems like useful information

Reply:
same